src/Controls/tests/UITests/Tests/Issues/Issue21846.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue21846 : _IssuesUITest
+	{
+		public Issue21846(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Fix crash closing Popup with WebView";
+
+		[Test]
+		[Category(UITestCategories.WebView)]
+		public void WebViewNoCrashPopup()
+		{
+			this.IgnoreIfPlatforms([TestDevice.Android, TestDevice.Mac, TestDevice.Windows]);
```

Comment:
I think this should be fine to run on Mac?

Reply:
No tested, I am going to do some tests but I think so, yes.